pons/routing/router.py

```diff
@@ -98,10 +103,28 @@ def on_scan_received(self, msg: pons.Message, remote_id: int):
             # self.log("DUP PEER: %d" % remote_id)
 
     def on_peer_discovered(self, peer_id):
-        self.log("peer discovered: %d" % peer_id)
+        #self.log("peer discovered: %d" % peer_id)
+        self.netsim.event_manager.on_peer_discovery(self.my_id, peer_id)
 
     def on_msg_received(self, msg: pons.Message, remote_id: int):
-        self.log("msg received: %s from %d" % (msg, remote_id))
+        self.netsim.routing_stats['relayed'] += 1
+        if not self.is_msg_known(msg):
+            self.remember(remote_id, msg)
```

Comment:
this is convinient for most routing algorithms but quite heavy for an empty router without logic.